eng/common/scripts/Update-ChangeLog.ps1

```diff
@@ -0,0 +1,109 @@
+# Note: This script will add or replace version title in change log
```

Comment:
I assume this script will replace https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/Update-Change-Log.ps1 and you will delete that once you transition folks to this one?

Reply:
Right